homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -211,6 +222,32 @@
         schema=SCHEMA_DEACTIVATE_VACATION,
     )
 
+    async def _set_active_climate_profile(service):
+        """Service to set the active climate profile."""
+        entity_id_list = service.data.get(ATTR_ENTITY_ID)
+        climate_profile_index = service.data.get(ATTR_CLIMATE_PROFILE_INDEX) - 1
+
+        for hapid in hass.data[DOMAIN]:
```

Comment:
```py
for hap in hass.data[DOMAIN].values():
    ...
```

Reply:
done. Also changed for the other services.